File: baselib/src/main/java/com/qisan/baselib/wigget/verticaltablayout/adapter/TabAdapter.java
package com.qisan.baselib.wigget.verticaltablayout.adapter;


import com.qisan.baselib.wigget.verticaltablayout.widget.TabView;

/**
 * @author chqiu
 *         Email:qstumn@163.com
 */
public interface TabAdapter {
    int getCount();

    TabView.TabBadge getBadge(int position);

    TabView.TabIcon getIcon(int position);

    TabView.TabTitle getTitle(int position);

    int getBackground(int position);
}

File: baselib/src/main/java/com/qisan/baselib/widget/verticaltablayout/adapter/TabAdapter.kt
package com.qisan.baselib.widget.verticaltablayout.adapter

import com.qisan.baselib.widget.verticaltablayout.widget.ITabView

/**
 * Created by QiSan 2022/6/17
 * package com.qisan.wanandroid.widget.verticaltablayout.adapter
 */
interface TabAdapter {

    fun getCount(): Int

    fun getBadge(position: Int): ITabView.TabBadge?

    fun getIcon(position: Int): ITabView.TabIcon?

    fun getTitle(position: Int): ITabView.TabTitle?

    fun getBackground(position: Int): Int

}